languages/csharp/AppEncryption/AppEncryption.IntegrationTests/Regression/AppEncryptionParameterizedTest.cs

```diff
@@ -140,5 +149,88 @@ public AppEncryptionParameterizedTest()
                     Times.Never);
             }
         }
+
+        private class AppEncryptionParameterizedTestData : IEnumerable<object[]>
+        {
+            private static readonly Random Random = new Random();
+            private readonly ConfigFixture configFixture;
+
+            public AppEncryptionParameterizedTestData()
+            {
+                configFixture = new ConfigFixture();
+                IMetricsRoot metrics = new MetricsBuilder()
```

Comment:
minor: be good to keep the old comment (or something even more descriptive) as to why this is needed